tpm/tpm_test.go

```diff
@@ -226,34 +226,6 @@ func TestSeal(t *testing.T) {
 	}
 }
 
-func TestReseal(t *testing.T) {
```

Comment:
Add TestReseal() and TestForceClear() back in. Or if necessary, remove them in a separate PR explaining the reason for their removal.

Reply:
Incorrect merge. Will fix.